src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -80,20 +82,36 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
     </div>
 
     <hr/>
-
+   
+	<script src="http://localhost:8080/js/TextEditor.js"></script>
     <% if (request.getSession().getAttribute("user") != null) { %>
-    <form action="/chat/<%= conversation.getTitle() %>" method="POST">
-        <input type="text" name="message">
+    <form action="/chat/<%= conversation.getTitle() %>" id = "form" method="POST">
+   		 <p>
+    		<input type="button" id="bBtn" value="B" style="font-weight:bold" onclick="setBold('editor');" />
```

Comment:
a shadow on the button would be good to know if bold is set for typing at that moment in time

Reply:
Tried a couple of ways but didn't work out when I choose multiple tools at once, will be on the TODO list.